### DISCUSSION TITLE
Set up repository

### DISCUSSION THREAD

**Discussion by PaulWAyers** [theochem/matrix-permanent#1] [Announcements]
@cassmasschelein and @msricher I set up a (public) repo for the matrix permanent. We need to keep this public so that Cassandra's instructor can access it, but I don't think that's an issue. @gabrielasd I also added you to it, as it will be useful/relevant for your work, and you might like to follow along.

@cassmasschelein and @msricher I'm going to "watch" this but I'll let you guys take the lead. @msricher a good first step could be to upload some notes, and perhaps an initial commit can be your old code (if that is useful). 

Please use discussions/issues for the repository, and I'll try to keep abreast of progress.

**Comment by msricher** [comment]
Added some notes and example code, and a link to the other code I don't have examples of (in David's fanpy repo). See the README.

**Reply by msricher** [reply]
One more thing I should add is my own thoughts on the problem. My intuition tells me that the combinatoric algorithm is best for some small values of M and N; then, the Glynn algorithm should be the best algorithm for large square matrices and slightly rectangular matrices (i.e. where M and N are close in value), and the Ryser algorithm should be best for large highly rectangular matrices (i.e. where the difference between M and N is large).
